src/pages/MainLayout.vue

```diff
@@ -15,17 +15,17 @@
         </template>
         <MainAlerts />
         <div class="mainContent row justify-between no-wrap">
-          <div class="whiteSpace gt-sm desktop-only"/>
-            <router-view class="desktop-only sidenav-desktop" name="sidenav"></router-view>
+          <div v-if="!$q.platform.is.mobile" class="whiteSpace gt-sm"/>
+            <router-view class="sidenav-desktop" name="sidenav"></router-view>
```

Comment:
Just checked out the branch - the router view (whyever) also needs the `v-if="!$q.platform.is.mobile"`, otherwise it will still display the sidenav on mobile for me.

Reply:
ah interesting! Seems to be different than a regular component or slot then..

Reply:
Ok now I get it.. I totally missed that the div above was self-closing, because router-view has child indentation. I really hope we get #680 soon...

Reply:
Divs for spacing also makes me feel quite uneasy